chunky/src/java/se/llbit/chunky/block/MinecraftBlockProvider.java

```diff
@@ -2653,6 +2653,11 @@ private static Block chest(Tag tag, boolean trapped) {
     return new Chest(name, type, facing, trapped);
   }
 
+  private static Block chain(Tag tag, String name, Texture texture) {
+    String axis = tag.get("Properties").get("axis").stringValue("north");
```

Comment:
The default value for `axis` is not `north` but `y`.

Reply:
I guess that's what I get for copy/pasting code instead of retyping it. lol